README.md

```diff
@@ -678,10 +730,133 @@ my_page(factory.get('/my_page?test=true'))
 output:
 
 ```shell
-{"function": "my_page", "level": "INFO", "logger": "your_logger", "message": "My page requested", "module": "<stdin>", "process": 20421, "request": {"method": "GET", "path": "/my_page", "headers": {"Cookie": ""}}, "response": {"success": true}, "thread": 140433370822464, "time": "2020-10-12T16:44:45.374508+02:00"}
+{
+  "function": "my_page", 
+  "level": "INFO", 
+  "logger": "your_logger", 
+  "message": "My page requested", 
+  "module": "<stdin>", 
+  "process": 20421, 
+  "request": {
+    "method": "GET", 
+    "path": "/my_page", 
+    "headers": {
+      "Cookie": ""
+    }
+  }, 
+  "response": {
+    "success": true
+  }, 
+  "thread": 140433370822464, 
+  "time": "2020-10-12T16:44:45.374508+02:00"
+}
+```
+
+### Case 6. Add parts of Django Request to JSON Output
+
+config.yaml
+
+```yaml
+version: 1
+
+root:
+  handlers:
+    - console
+  level: DEBUG
+  propagate: True
+
+filters:
+  isotime:
+    (): logging_utilities.filters.TimeAttribute
+  django:
+    (): logging_utilities.filters.django_request.JsonDjangoRequest
+    include_keys:
+      - request.path
+      - request.method
+      - request.headers
+
+formatters:
+  json:
+    class: logging_utilities.formatters.json_formatter.JsonFormatter
+    format:
+      time: isotime
+      level: levelname
+      logger: name
+      module: module
+      function: funcName
+      process: process
+      thread: thread
+      request_path: request.path
+      request_method: request.method
```

Comment:
maybe change `request_method: request.method` to `request.method: request.method` and add a comment to show that the name of the field can be a (random) string, containing (any?) special character.